### ISSUE TITLE
Segmentation fault (core dumped) in Flask APP

### ISSUE THREAD

**Issue by bharatk101-dckap** [pytorch/pytorch#32765]
## Segmentation fault (core dumped) after implementing a PyTorch model in  Flask APP

Pytorch Version - '1.4.0+cpu'

I've built a flask app that takes a sentence and predicts it's sentiment.

Here's the debug message/error

```
Reading symbols from python...done.
(gdb) r manage.py 
Starting program: /home/dckap/Projects/devenv/bin/python manage.py
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[Detaching after fork from child process 7658]
[New Thread 0x7ffff50e9700 (LWP 7660)]
[New Thread 0x7ffff4875700 (LWP 7661)]
[New Thread 0x7fffed983700 (LWP 7665)]
[New Thread 0x7fffed182700 (LWP 7666)]
[New Thread 0x7fffec981700 (LWP 7667)]

Thread 1 "python" received signal SIGSEGV, Segmentation fault.
BFD: reopening /tmp/tmp2c03b_40/stanfit4anon_model_861b75c6337e237650a61ae58c4385ef_3424227499479756936.cpython-37m-x86_64-linux-gnu.so: No such file or directory

BFD: reopening /tmp/tmp2c03b_40/stanfit4anon_model_861b75c6337e237650a61ae58c4385ef_3424227499479756936.cpython-37m-x86_64-linux-gnu.so: No such file or directory

BFD: reopening /tmp/tmp2c03b_40/stanfit4anon_model_861b75c6337e237650a61ae58c4385ef_3424227499479756936.cpython-37m-x86_64-linux-gnu.so: No such file or directory

warning: Can't read data for section '.eh_frame' in file '/tmp/tmp2c03b_40/stanfit4anon_model_861b75c6337e237650a61ae58c4385ef_3424227499479756936.cpython-37m-x86_64-linux-gnu.so'
0x00007fffb301de8d in pybind11::detail::make_new_python_type(pybind11::detail::type_record const&) () from /home/dckap/Projects/devenv/lib/python3.7/site-packages/torch/lib/libtorch_python.so
(gdb) where
#0  0x00007fffb301de8d in pybind11::detail::make_new_python_type(pybind11::detail::type_record const&) () from /home/dckap/Projects/devenv/lib/python3.7/site-packages/torch/lib/libtorch_python.so
#1  0x00007fffb30288e6 in pybind11::detail::generic_type::initialize(pybind11::detail::type_record const&) () from /home/dckap/Projects/devenv/lib/python3.7/site-packages/torch/lib/libtorch_python.so
#2  0x00007fffb31e489e in torch::onnx::initONNXBindings(_object*) () from /home/dckap/Projects/devenv/lib/python3.7/site-packages/torch/lib/libtorch_python.so
#3  0x00007fffb2f496d5 in initModule() () from /home/dckap/Projects/devenv/lib/python3.7/site-packages/torch/lib/libtorch_python.so
#4  0x0000555555786595 in _PyImport_LoadDynamicModuleWithSpec (spec=0x7fffb3c75198, fp=<optimized out>) at /tmp/build/80754af9/python_1553721932202/work/Python/importdl.c:159
#5  0x00005555557867a9 in _imp_create_dynamic_impl.isra.15 (file=0x0, spec=0x7fffb3c75198) at /tmp/build/80754af9/python_1553721932202/work/Python/import.c:2158
#6  _imp_create_dynamic (module=<optimized out>, args=<optimized out>, nargs=<optimized out>) at /tmp/build/80754af9/python_1553721932202/work/Python/clinic/import.c.h:289
#7  0x00005555556909e2 in _PyMethodDef_RawFastCallDict (method=0x5555558837a0 <imp_methods+320>, self=<optimized out>, args=0x7fffb3effcd8, nargs=1, kwargs=0x7fffb3f04b40)
    at /tmp/build/80754af9/python_1553721932202/work/Objects/call.c:530
#8  0x0000555555690ae1 in _PyCFunction_FastCallDict (func=0x7ffff6dd7d80, args=<optimized out>, nargs=<optimized out>, kwargs=<optimized out>)
    at /tmp/build/80754af9/python_1553721932202/work/Objects/call.c:582
#9  0x000055555572a151 in do_call_core (kwdict=0x7fffb3f04b40, callargs=0x7fffb3effcc0, func=0x7ffff6dd7d80) at /tmp/build/80754af9/python_1553721932202/work/Python/ceval.c:4641
#10 _PyEval_EvalFrameDefault (f=<optimized out>, throwflag=<optimized out>) at /tmp/build/80754af9/python_1553721932202/work/Python/ceval.c:3191
#11 0x000055555566a4f9 in _PyEval_EvalCodeWithName (_co=0x7ffff6df6db0, globals=<optimized out>, locals=<optimized out>, args=<optimized out>, argcount=<optimized out>, kwnames=0x0, 
    kwargs=0x7ffff6c94740, kwcount=0, kwstep=1, defs=0x0, defcount=0, kwdefs=0x0, closure=0x0, name=0x7ffff6dbbb70, qualname=0x7ffff6dbbb70)
    at /tmp/build/80754af9/python_1553721932202/work/Python/ceval.c:3930
--Type <RET> for more, q to quit, c to continue without paging--

```

cc @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by mrshenli** [comment]
@bharatk101-dckap could you please share the minimum code for to reproduce the error? And what is your dev env?

**Comment by mrshenli** [comment]
Adding onnx label due to the following error trace:

```
#2  0x00007fffb31e489e in torch::onnx::initONNXBindings(_object*) () from /home/dckap/Projects/devenv/lib/python3.7/site-packages/torch/lib/libtorch_python.so
```

**Comment by bharatk101-dckap** [comment]
> @bharatk101-dckap could you please share the minimum code for to reproduce the error? And what is your dev env?

Dev Env - Ubuntu 18.10, Python 3.7, Torch - 1.4.0+cpu, torchtext - 0.5.0, Flask - 1.1.1

To reproduce this error creat a flask app and load a pre-trained model in a separate file and import that model in flask app.

**Comment by zjmtlab** [comment]
Any solution or workaround for this issue? Seems I meet same issue with similar env. Thanks very much!

**Comment by garymm** [comment]
@bharatk101-dckap is this still an issue on latest PyTorch nightly?
If so, could you please provide some repro code?

**Comment by AllenTiTaiWang** [comment]
Hi @bharatk101-dckap ,

We’ve gone ahead and closed this issue because it is stale.
If you still believe this issue is relevant, please feel free to reopen the issue and we will triage it as necessary. Please specify in a comment any updated information you may have so that we can address it effectively. We encourage you to try the latest pytorch-preview (nightly) version to see if it has resolved the issue.

Thanks,
ONNX Converter team
